packer.mk

```diff
@@ -60,20 +60,21 @@ else
 endif
 
 build-packer-windows:
-ifeq (${MODE},sigMode)
+ifeq (${MODE},windowsVhdMode)
+ifeq (${SIG_FOR_PRODUCTION},True)
 ifeq (${HYPERV_GENRATION},V1)
-	@echo "${MODE}: Building with Hyper-v generation 1 VM and save to Shared Image Gallery"
+	@echo "${MODE}: Building with Hyper-v generation 1 VM and save to Classic Storage Account"
 else
-ifeq (${GEN2_SIG_FOR_PRODUCTION},True)
 	@echo "${MODE}: Building with Hyper-v generation 2 VM and save to Classic Storage Account"
+endif
+else
+ifeq (${HYPERV_GENRATION},V1)
+	@echo "${MODE}: Building with Hyper-v generation 1 VM and save to Shared Image Gallery"
 else
-	@echo "${MODE}: Building and save to Shared Image Gallery"
+	@echo "${MODE}: Building with Hyper-v generation 2 VM and save to Shared Image Gallery"
 endif
 endif
 	@packer build -var-file=vhdbuilder/packer/settings.json vhdbuilder/packer/windows-vhd-builder-sig.json
```

Comment:
Should we remove this line `@packer build -var-file=vhdbuilder/packer/settings.json vhdbuilder/packer/windows-vhd-builder-sig.json` and the file `vhdbuilder/packer/settings.json vhdbuilder/packer/windows-vhd-builder-sig.json`?